src/test/java/se/sundsvall/incident/integration/db/entity/IncidentEntityTest.java

```diff
@@ -0,0 +1,188 @@
+		assertThat(updated.getAnnotations()).hasSize(1);
+		assertThat(updated.getType()).isEqualTo(LocalDateTime.class);
+
+		final Column column = updated.getDeclaredAnnotation(Column.class);
+		assertThat(column.name()).isEqualTo("updated");
+	}
+
+	@Test
+	void testDescriptionHasCorrectAnnotationsAndValues() {
+		final Field description = FieldUtils.getDeclaredField(IncidentEntity.class, "description", true);
+		assertThat(description.getAnnotations()).hasSize(1);
+		assertThat(description.getType()).isEqualTo(String.class);
+
+		final Column column = description.getDeclaredAnnotation(Column.class);
+		assertThat(column.name()).isEqualTo("description");
+	}
+
+	@Test
+	void testMapCoordinatesHasCorrectAnnotationsAndValues() {
+		final Field mapCoordinates = FieldUtils.getDeclaredField(IncidentEntity.class, "coordinates", true);
+		assertThat(mapCoordinates.getAnnotations()).hasSize(1);
+		assertThat(mapCoordinates.getType()).isEqualTo(String.class);
+
+		final Column column = mapCoordinates.getDeclaredAnnotation(Column.class);
+		assertThat(column.name()).isEqualTo("coordinates");
+	}
+
+	@Test
+	void testFeedbackHasCorrectAnnotationsAndValues() {
+		final Field feedback = FieldUtils.getDeclaredField(IncidentEntity.class, "feedback", true);
+		assertThat(feedback.getAnnotations()).hasSize(1);
+		assertThat(feedback.getType()).isEqualTo(String.class);
+
+		final Column column = feedback.getDeclaredAnnotation(Column.class);
+		assertThat(column.name()).isEqualTo("feedback");
+	}
+
+	@Test
+	void testStatusHasCorrectAnnotationsAndValues() {
+		final Field status = FieldUtils.getDeclaredField(IncidentEntity.class, "status", true);
+		assertThat(status.getAnnotations()).hasSize(2);
+		assertThat(status.getType()).isEqualTo(Status.class);
+
+		final Enumerated enumerated = status.getDeclaredAnnotation(Enumerated.class);
+		final Column column = status.getDeclaredAnnotation(Column.class);
+		assertThat(enumerated.value()).isEqualTo(EnumType.STRING);
+		assertThat(column.name()).isEqualTo("status");
+	}
+
+	@Test
+	void testCategoryHasCorrectAnnotationsAndValues() {
+		final Field category = FieldUtils.getDeclaredField(IncidentEntity.class, "category", true);
+		assertThat(category.getAnnotations()).hasSize(2);
+		assertThat(category.getType()).isEqualTo(Category.class);
+
+		final Enumerated enumerated = category.getDeclaredAnnotation(Enumerated.class);
+		final Column column = category.getDeclaredAnnotation(Column.class);
+		assertThat(enumerated.value()).isEqualTo(EnumType.STRING);
+		assertThat(column.name()).isEqualTo("category");
+	}
+
+	@Test
+	void testBean() {
+		MatcherAssert.assertThat(IncidentEntity.class, allOf(
+			hasValidBeanConstructor(),
+			hasValidGettersAndSetters()));
+	}
+
+	@Test
+	void preUpdateTest() {
+		var incident = buildIncidentEntity(Category.VATTENMATARE);
+		incident.preUpdate();
+
+		assertThat(incident.getUpdated()).isNotNull();
+		assertThat(incident.getUpdated()).isAfterOrEqualTo(LocalDateTime.now().minusSeconds(2));
```

Comment:
The line below seems to be a more straightforward way of doing this:
`assertThat(incident.getUpdated()).isCloseTo(now(systemDefault()), within(2, SECONDS));`